pom.xml

```diff
@@ -280,15 +277,15 @@
                 </executions>
             </plugin>
             <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-jar-plugin</artifactId>
-              <configuration>
-                <archive>
-                  <manifestEntries>
-                    <Automatic-Module-Name>${java-module-name}</Automatic-Module-Name>
-                  </manifestEntries>
-                </archive>
-              </configuration>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Note that this is already configured by Spring Data Build Parent.

Reply:
Ok. Removed.